### PR TITLE
Update to PHPUnit 9 and Mantle Testkit

### PR DESCRIPTION
**Pull request by srtfisher**
- Updates to PHPUnit 9 and Mantle Testkit
- Gets Github Actions to pass again

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -1,23 +1,44 @@
 {
     "name": "alleyinteractive/wp-asset-manager",
+    "license": "GPL-2.0-only",
+    "type": "wordpress-plugin",
```
Comment (kevinfodness) [inline composer.json:4]: 🙌

File: composer.json [hunk 2]
```diff
@@ -1,23 +1,44 @@
...
+            "dealerdirect/phpcodesniffer-composer-installer": true,
+            "alleyinteractive/composer-wordpress-autoloader": true
+        },
+        "sort-packages": true
     },
+    "prefer-stable": true,
     "scripts": {
-        "phpcs" : "phpcs --standard=./phpcs.xml .",
-        "phpcbf" : "phpcbf ."
+        "lint": "@phpcs",
```
Comment (kevinfodness) [inline composer.json:37]: This is nice.

File: php/class-asset-manager.php [hunk 3]
```diff
@@ -520,7 +520,7 @@ public function asset_should_add( $asset ) {
 			$condition_include = ! is_array( $condition_include ) ? [ $condition_include ] : $condition_include;
 
 			foreach ( $condition_include as $condition_true ) {
-				if ( $conditions[ $condition_true ] ) {
+				if ( isset( $conditions[ $condition_true ] ) && $conditions[ $condition_true ] ) {
```
Comment (kevinfodness) [inline php/class-asset-manager.php]: Why not just do
```suggestion
				if ( ! empty( $conditions[ $condition_true ] ) ) {
```

**Review by kevinfodness — APPROVED** [review]
🍣

File: php/class-asset-manager.php (hunk 3 above)
Comment (srtfisher) [reply php/class-asset-manager.php]: hahahahahahahaaaaa, oh man...

**Pull request merged**